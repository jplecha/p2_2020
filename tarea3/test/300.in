# Prueba menores
insertarEnBinario (6,5.0)
insertarEnBinario (3,4.1)
insertarEnBinario (9,7.2)
insertarEnBinario (2,1.4)
insertarEnBinario (5,2.6)
insertarEnBinario (8,5.1)
insertarEnBinario (11,7.1)
insertarEnBinario (1,8.2)
insertarEnBinario (4,0.7)
insertarEnBinario (7,6.8)
insertarEnBinario (10,1.1)
menores 5
reiniciar
insertarEnBinario (6,5.0)
insertarEnBinario (3,4.1)
insertarEnBinario (9,7.2)
insertarEnBinario (2,6.4)
insertarEnBinario (5,2.6)
insertarEnBinario (8,5.1)
insertarEnBinario (11,7.1)
insertarEnBinario (1,8.2)
insertarEnBinario (4,0.7)
insertarEnBinario (7,6.8)
insertarEnBinario (10,1.1)
menores 5
reiniciar
insertarEnBinario (6,5.0)
insertarEnBinario (3,4.1)
insertarEnBinario (9,7.2)
insertarEnBinario (2,1.4)
insertarEnBinario (5,2.6)
insertarEnBinario (8,5.1)
insertarEnBinario (11,2.1)
insertarEnBinario (1,8.2)
insertarEnBinario (4,0.7)
insertarEnBinario (7,6.8)
insertarEnBinario (10,1.1)
menores 5
reiniciar
insertarEnBinario (6,5.0)
insertarEnBinario (3,4.1)
insertarEnBinario (9,7.2)
insertarEnBinario (2,1.4)
insertarEnBinario (5,2.6)
insertarEnBinario (8,5.1)
insertarEnBinario (11,7.1)
insertarEnBinario (1,8.2)
insertarEnBinario (4,0.7)
insertarEnBinario (7,6.8)
insertarEnBinario (10,1.1)
insertarEnBinario (12,2.1)
menores 5
reiniciar
insertarEnBinario (6,5.0)
insertarEnBinario (3,4.1)
insertarEnBinario (9,7.2)
insertarEnBinario (2,1.4)
insertarEnBinario (5,2.6)
insertarEnBinario (8,5.1)
insertarEnBinario (12,7.1)
insertarEnBinario (1,8.2)
insertarEnBinario (4,0.7)
insertarEnBinario (7,6.8)
insertarEnBinario (10,1.1)
insertarEnBinario (16,0.7)
insertarEnBinario (11,1.8)
insertarEnBinario (15,1.1)
menores 5
Fin