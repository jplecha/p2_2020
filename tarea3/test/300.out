1>#  Prueba menores.
2>Insertado.
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>Insertado.
8>Insertado.
9>Insertado.
10>Insertado.
11>Insertado.
12>Insertado.
13>
-(10,1.10)
(5,2.60)
--(4,0.70)
-(3,4.10)
--(2,1.40)
14>Estructuras reiniciadas.
15>Insertado.
16>Insertado.
17>Insertado.
18>Insertado.
19>Insertado.
20>Insertado.
21>Insertado.
22>Insertado.
23>Insertado.
24>Insertado.
25>Insertado.
26>
-(10,1.10)
(5,2.60)
--(4,0.70)
-(3,4.10)
27>Estructuras reiniciadas.
28>Insertado.
29>Insertado.
30>Insertado.
31>Insertado.
32>Insertado.
33>Insertado.
34>Insertado.
35>Insertado.
36>Insertado.
37>Insertado.
38>Insertado.
39>
-(11,2.10)
--(10,1.10)
(5,2.60)
--(4,0.70)
-(3,4.10)
--(2,1.40)
40>Estructuras reiniciadas.
41>Insertado.
42>Insertado.
43>Insertado.
44>Insertado.
45>Insertado.
46>Insertado.
47>Insertado.
48>Insertado.
49>Insertado.
50>Insertado.
51>Insertado.
52>Insertado.
53>
--(12,2.10)
-(10,1.10)
(5,2.60)
--(4,0.70)
-(3,4.10)
--(2,1.40)
54>Estructuras reiniciadas.
55>Insertado.
56>Insertado.
57>Insertado.
58>Insertado.
59>Insertado.
60>Insertado.
61>Insertado.
62>Insertado.
63>Insertado.
64>Insertado.
65>Insertado.
66>Insertado.
67>Insertado.
68>Insertado.
69>
--(16,0.70)
---(15,1.10)
-(11,1.80)
--(10,1.10)
(5,2.60)
--(4,0.70)
-(3,4.10)
--(2,1.40)
70>Fin.
